keyboards/keychron/k3/keymaps/iso/keymap.c

```diff
@@ -21,13 +21,25 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 // Layer names don't all need to be of the same length, obviously, and you can also skip them
 // entirely and just use numbers.
 enum layer_names {
-  _BASE = 0,
-  _FN,
+  MAC_BASE = 0,
+  WIN_BASE,
+  MAC_FM,
```

Comment:
_FM is a weird choice for a FN layer. Any specific reason why?

Reply:
urgh, just a typo, should be MAC_FN and WIN_FN